src/components/Compose.js

```diff
@@ -22,6 +22,7 @@ function Compose(props) {
     onBeforePost,
     postFile,
     addNotification,
+    lastMessageText,
```

Comment:
I think it should live in `state`, and `updateState` should be used for storing the last message.